lib/Wikia/src/Logger/WebProcessor.php

```diff
@@ -62,7 +62,13 @@ public function popContext($type=self::RECORD_TYPE_CONTEXT) {
 	 * @return array list of context objects for the given $type
 	 */
 	private function prepareContext($type) {
-		$this->sharedContext[$type] = call_user_func_array('array_merge', array_reverse($this->contextStack[$type]));
+		$stack = array_reverse($this->contextStack[$type]);
```

Comment:
maybe we should use push this reverse logic into pushContext() and popContext() and use:
* [] operator instead of array_unshift()
* array_pop() instead of array_shift()
?